sdk/paloaltonetworksngfw/arm-paloaltonetworksngfw/CHANGELOG.md

```diff
@@ -1,21 +1,86 @@
+  - Added Interface PaloAltoNetworksCloudngfwOperationsCreateProductSerialNumberOptionalParams
+  - Added Interface PaloAltoNetworksCloudngfwOperationsListCloudManagerTenantsOptionalParams
+  - Added Interface PaloAltoNetworksCloudngfwOperationsListProductSerialNumberStatusOptionalParams
+  - Added Interface PaloAltoNetworksCloudngfwOperationsListSupportInfoOptionalParams
+  - Added Interface PostRulesCreateOrUpdateHeaders
+  - Added Interface PostRulesDeleteHeaders
+  - Added Interface PrefixListGlobalRulestackCreateOrUpdateHeaders
+  - Added Interface PrefixListGlobalRulestackDeleteHeaders
+  - Added Interface PrefixListLocalRulestackCreateOrUpdateHeaders
+  - Added Interface PrefixListLocalRulestackDeleteHeaders
+  - Added Interface PreRulesCreateOrUpdateHeaders
+  - Added Interface PreRulesDeleteHeaders
+  - Added Interface ProductSerialNumberRequestStatus
+  - Added Interface ProductSerialNumberStatus
+  - Added Interface StrataCloudManagerConfig
+  - Added Interface StrataCloudManagerInfo
+  - Added Interface SupportInfoModel
+  - Interface FirewallResource has a new optional parameter isStrataCloudManaged
+  - Interface FirewallResource has a new optional parameter strataCloudManagerConfig
+  - Interface FirewallResourceUpdateProperties has a new optional parameter isStrataCloudManaged
+  - Interface FirewallResourceUpdateProperties has a new optional parameter strataCloudManagerConfig
+  - Interface FirewallStatusResource has a new optional parameter isStrataCloudManaged
+  - Interface FirewallStatusResource has a new optional parameter strataCloudManagerInfo
+  - Interface NetworkProfile has a new optional parameter privateSourceNatRulesDestination
+  - Added Type Alias EnableStatus
+  - Added Type Alias GlobalRulestackCommitResponse
+  - Added Type Alias LocalRulestacksCommitResponse
+  - Added Type Alias LocalRulestacksListAppIdsNextResponse
+  - Added Type Alias LocalRulestacksListCountriesNextResponse
+  - Added Type Alias LocalRulestacksListPredefinedUrlCategoriesNextResponse
+  - Added Type Alias MetricsObjectFirewallCreateOrUpdateResponse
+  - Added Type Alias MetricsObjectFirewallDeleteResponse
+  - Added Type Alias MetricsObjectFirewallGetResponse
+  - Added Type Alias MetricsObjectFirewallListByFirewallsNextResponse
+  - Added Type Alias MetricsObjectFirewallListByFirewallsResponse
+  - Added Type Alias PaloAltoNetworksCloudngfwOperationsCreateProductSerialNumberResponse
+  - Added Type Alias PaloAltoNetworksCloudngfwOperationsListCloudManagerTenantsResponse
+  - Added Type Alias PaloAltoNetworksCloudngfwOperationsListProductSerialNumberStatusResponse
+  - Added Type Alias PaloAltoNetworksCloudngfwOperationsListSupportInfoResponse
+  - Added Type Alias ProductSerialStatusValues
+  - Added Type Alias RegistrationStatus
+  - Added Enum KnownEnableStatus
+  - Added Enum KnownRegistrationStatus
 
 ### Breaking Changes
+  - Operation GlobalRulestack.beginCommitAndWait has a new signature
```

Comment:
why we have this breaking?

Reply:
@kazrael2119 Could you add directives to remove this breaking? I don't think it is necessary.